src/components/button/style.ts

```diff
@@ -53,6 +53,11 @@ export const StyledButton = styled.button<{
     padding: 10px 16px 10px 22px;
   }
 
+  &.focusDisabled:focus:not(:focus-visible) {
```

Comment:
no need to do it, instead of it you should update `:focus` and probably `:active` subclasses to` :focus-visible` and it will solve your problem

Reply:
indeed it works like that too. fixed